docs/developer/plugin-list.asciidoc

```diff
@@ -452,7 +452,7 @@ the infrastructure monitoring use-case within Kibana.
 
 
 |{kib-repo}blob/{branch}/x-pack/plugins/ingest_pipelines/README.md[ingestPipelines]
-|The ingest_pipelines plugin provides Kibana support for Elasticsearch's ingest nodes. Please refer to the Elasticsearch documentation for more details.
+|The ingest_pipelines plugin provides Kibana support for Elasticsearch's ingest pipelines.
```

Comment:
do you also want to change this line to use {kib} and {es}?

Reply:
This file gave me a lot of CI trouble. 😄 

I didn't realize it at first, but this section is generated from x-pack/plugins/ingest_pipelines/README.md:
https://github.com/elastic/kibana/blob/7197fef216e72242087f721a0486d612862ad5bd/x-pack/plugins/ingest_pipelines/README.md#L4

Since that file is `.md`, attributes aren't supported.